src/utils/graph/constraints.js

```diff
@@ -1,4 +1,4 @@
-import { distance1d, greaterOrEqual, equalTo, subtract } from './common';
+import { Constraint, Operator, Strength } from 'kiwi.js';
 
 /**
  * Constraint base definitions.
```

Comment:
Do you mind expanding on this comment a little to explain what this means?

Reply:
There will be some follow up work coming soon which might change this up a bit, but then I think it will be stable enough to get documenting, if that's reasonable?

Reply:
ok, but it might be worth adding a JIRA ticket for these subsequent documentation improvements if they're out of scope for this PR